notebooks/Perception_Statistics.ipynb

```diff
@@ -13,11 +13,11 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 8,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "data_root = \"/data/<GUID>\""
+    "data_root = \"/Users/sanjay.vishwakarma/data_test\""
```

Comment:
Please clean the notebook (remove the personal path and all outputs) in this PR.